package.json

```diff
@@ -34,38 +42,42 @@
   },
   "homepage": "https://github.com/mattlewis92/webpack-filter-warnings-plugin#readme",
   "files": [
-    "dist"
+    "cjs",
+    "es"
   ],
   "engines": {
     "node": ">= 4.3 < 5.0.0 || >= 5.10"
   },
   "peerDependencies": {
+    "@types/webpack": "^4.4.25",
```

Comment:
Let's remove this and stick with documenting it as required in the readme, if you're not using typescript then it's unfair to force users to install typings they don't need

Reply:
As of npm@3 this does not get installed, it only triggers warning during installation

Reply:
People will still open issues complaining about it though, sadly npm doesn't have optional peer dependencies 😢